src/_posts/2020-09-17-cassandra-usage-report-2020.markdown

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "Cassandra Usage Report 2020"
+date:   2020-09-17 09:00:00 -0700
+author: the Apache Cassandra Community
+categories: blog
+---
+
+Apache Cassandra is the open source NoSQL database for mission critical data. Today the community announced findings from a comprehensive global survey of 901 practitioners on Cassandra usage. It’s the first of what will become an annual survey that provides a baseline understanding of who, how, and why organizations use Cassandra.
+
+
+    "I saw zero downtime at global scale with Apache Cassandra. That's a powerful statement to make. For our business that’s quite crucial." - Practitioner, London
+
+**Cassandra adoption is correlated with organizations in a more advanced stage of digital transformation.**
```

Comment:
Do you want to consider adding a section header for clarity in the reading flow ```Executive Summary ``` or ``` Themes```?